_pages/technical-details.markdown

```diff
@@ -0,0 +1,69 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: single
+#title: Technical Details
+permalink: /technical-info
+header:
+  image: /assets/images/data-model-header.jpg
+  caption: "Photo credit: [**Toby56**](https://unsplash.com/@toby56?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
+---
+
+TermIt is an information system with a strong formal background. This page contains some technical background of TermIt. 
```

Comment:
"TermIt is an information system with a strong formal background."

Does not say much IMO, would remove it.